src/components/Dashboard/TodoList.vue

```diff
@@ -181,64 +172,45 @@
                   <v-btn text loading disabled />
                 </template>
 
-                <!-- draft correction or alteration or conversion (staff only) -->
-                <template v-else-if="isStatusDraft(item)
-                  && (isTypeCorrection(item) || isTypeAlteration(item) || isTypeConversion(item))
```

Comment:
This section no longer includes draft alterations, which are now handled in the "other draft filing" section below.